<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForGraspLabel</class>
 <widget class="QMainWindow" name="ForGraspLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPM - Grasp Path Marker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,2">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="9,0,0,1">
         <item>
          <widget class="QWidget" name="graphWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1">
            <item>
             <widget class="QLabel" name="graphWindow">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">border:1px solid rgb(0,0,0);background-color: white</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Classes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="axisLabel">
             <property name="text">
              <string>()</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="classList">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="flow">
            <enum>QListView::LeftToRight</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0,0,0,1,0,8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>    No:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="stepNum">
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPushButton" name="lastButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>last</string>
              </property>
              <property name="shortcut">
               <string>[</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="nextButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>next</string>
              </property>
              <property name="shortcut">
               <string>]</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="saveButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+S</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="autosaveCheck">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Autosave</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+Shift+S</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Scoll Speed</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="scollSpeedSlider">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Jaw Width</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="jawWidthInput">
                 <property name="focusPolicy">
                  <enum>Qt::ClickFocus</enum>
                 </property>
                 <property name="text">
                  <string>30</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="originWidthCheck">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Save Origin Object Width</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QRadioButton" name="simpleWidthRadio">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Width</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="complexWidthRadio">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Width(array)</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QPushButton" name="redoButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Redo</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+Y</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="undoButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Undo</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+Z</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="clearButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+C</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="deleteButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+D</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Grasp labels:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="labelList">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionDataset_Path"/>
    <addaction name="actionLabel_Path"/>
    <addaction name="actionBoth_Path"/>
    <addaction name="separator"/>
    <addaction name="actionClass_File"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="actionAutoResize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuview"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDataset_Path">
   <property name="text">
    <string>Dataset Path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionLabel_Path">
   <property name="text">
    <string>Label Path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="actionClass_File">
   <property name="text">
    <string>Class File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionBoth_Path">
   <property name="text">
    <string>Both Path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionAutoResize">
   <property name="text">
    <string>Auto Resize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
